### PR TITLE
Typescript bindings for points program

### PR DESCRIPTION
**Pull request by moshthepitt**
## Changes

This PR adds Typescript bindings for the Points program and Points Store program.

Both programs are in this companion PR: https://github.com/staratlasmeta/star-atlas-programs/pull/119

## Checklist

- [ ] UAT
- [ ] Passes final QA checks

### CODE REVIEW

**Comment by bravetarget** [comment]
Task linked: [CU-257en2b Factory calls for XP Program](https://app.clickup.com/t/257en2b)

File: src/points/instruction_builders/updatePointCategoryAccountIx.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        admin,
+        domainAccount: domainKey,
+        pointCategoryAccount: pointCategoryKey,
+      })
+      .remainingAccounts(remainingAccounts)
+      .instruction(),
+  ];
+
+  return {
+    signers: [],
```
Comment (turnyay) [inline src/points/instruction_builders/updatePointCategoryAccountIx.ts:96]: do we need this on every Ix? theyre all []

Comment (moshthepitt) [reply src/points/instruction_builders/updatePointCategoryAccountIx.ts]: Changed these to be optional

**Comment by moshthepitt** [comment]
This has been superseded and replaced by the new-style Typescript bindings here: https://github.com/staratlasmeta/star-atlas-programs/pull/119

Specifically:

- [Points Program](https://github.com/staratlasmeta/star-atlas-programs/tree/scream-develop/sol-programs/scream/tests/src/points)
- [Points Store Program](https://github.com/staratlasmeta/star-atlas-programs/tree/scream-develop/sol-programs/scream/tests/src/pointsStore)

**Pull request closed without merging**